apps/web/public/static/locales/en/common.json

```diff
@@ -835,6 +835,7 @@
   "please_enter_phone_number": "Please enter a phone number",
   "agent_updated_successfully": "Agent updated successfully",
   "agent_created_successfully": "Agent created successfully",
+  "agent_event_type_updated_successfully": "Event type added successfully",
```

Comment:
nit: Key name and actually translation are contradicting. 

Should it be added or updated?